core/indexer/processTransactions.go

```diff
@@ -254,6 +266,15 @@ func (tdp *txDatabaseProcessor) groupNormalTxsAndRewards(
 	return transactions, rewardsTxs
 }
 
+func (tdp *txDatabaseProcessor) shouldIndex(txHandler data.TransactionHandler) bool {
+	if !tdp.isInImportMode {
+		return true
+	}
+
+	destShardId := tdp.shardCoordinator.ComputeId(txHandler.GetRcvAddr())
```

Comment:
you dont need shardCoordinator here because you have destination shard from miniblock. no need to insert shardCoordinator here.

Reply:
will be fixed on the new elastic search implementation feature branch :+1: